hardhat.config.js

```diff
@@ -65,7 +117,7 @@ module.exports = {
       chainId: 43114,
       gasPrice: 225000000000,
       throwOnTransactionFailures: false,
-      loggingEnabled: true,
+      // loggingEnabled: true,
```

Comment:
This is optional, I noticed the output was clearer when using hardhat's network

Reply:
> This is optional, I noticed the output was clearer when using hardhat's network